### PR TITLE
Add more installation instructions to Readme

### LINKED ISSUE #245: Add alternative ways to install python-lsp-server
Currently in main README file, the only way listed to install it is - through pip, which is probably fine way, but it would be great to see more clean and elegant ways to install it, like through official linux repos (for example [fedora](https://src.fedoraproject.org/rpms/python-lsp-server), [debian](https://packages.debian.org/bookworm/python3-pylsp)/deb-based OSes or [arch](https://archlinux.org/packages/community/any/python-lsp-server/)), or maybe some people on windows can install it using [conda](https://anaconda.org/conda-forge/python-lsp-server)

If i can get a permission, i can also write it myself and submit it as PR.

### PR DESCRIPTION
**Pull request by nougcat**
Windows and Linux installation instructions are now listed in readme

Fixes #245

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,31 @@ If you get an error similar to `'install_requires' must be a string or list of s
 pip install -U setuptools
 ```
 
+### Windows and Linux installation
+
+You can install python lsp server on Windows using this conda command
```
Comment (ccordoba12) [inline README.md]:
```suggestion
If you use Anaconda/Miniconda, you can install `python-lsp-server` using this conda command

```

File: README.md [hunk 2]
```diff
@@ -43,6 +43,31 @@ If you get an error similar to `'install_requires' must be a string or list of s
...
+```
+conda install -c conda-forge python-lsp-server 
+```
+
+Python lsp server is available in repo of every major Linux distribution, and it is usually called `python-lsp-server`
```
Comment (ccordoba12) [inline README.md]:
```suggestion
Python-lsp-server is available in the repos of every major Linux distribution, and it is usually called `python-lsp-server`.
```

File: README.md [hunk 3]
```diff
@@ -43,6 +43,31 @@ If you get an error similar to `'install_requires' must be a string or list of s
...
+
+For example, here is how to install it in Debian and Debian-based distributions (E.g. Ubuntu, Pop!_OS, Linux Mint)
+
+```
+sudo apt-get install python-lsp-server
+```
+or Fedora Linux
```
Comment (ccordoba12) [inline README.md:62]:
```suggestion

or Fedora Linux
```

File: README.md [hunk 4]
```diff
@@ -43,6 +43,31 @@ If you get an error similar to `'install_requires' must be a string or list of s
...
+
+```
+sudo dnf install python-lsp-server
+```
+
+Only Alpine Linux package is named differently, you can install it by typing this command in your terminal
```
Comment (ccordoba12) [inline README.md]:
```suggestion
Only on Alpine Linux the package is named differently. You can install it there by typing this command in your terminal:

```

**Review by ccordoba12 — COMMENTED** [review]
Thanks @nougcat for your help with this! I left some suggestions for you, otherwise looks good to me.

**Comment by nougcat** [comment]
Thanks for suggestions @ccordoba12 , I added all your suggestions in the commit [bd67260](https://github.com/python-lsp/python-lsp-server/pull/248/commits/bd67260a18cf7a84d9c50216691cab52d7c07398)

Also I'm sorry for not responding to you, at the time I have committed your suggestions, I'm still learning how to collaborate with people on github.

**Review by ccordoba12 — APPROVED** [review]
Thanks a lot for your help @nougcat!

Note: The failures in our tests were fixed already in PR #258.

**Pull request merged**